tests/scripting/ScriptEngine.cxx

```diff
@@ -0,0 +1,13 @@
+#include <catch.hpp>
+#include "ScriptEngine.hxx"
+#include "Filesystem.hxx"
+
+TEST_CASE("The test script can be loaded", "[ScriptEngine]")
+{
+#ifdef USE_ANGELSCRIPT
```

Comment:
Since you already put the test under the `#ifdef` in tests/CMakeLists.txt, wouldn't this `#ifdef` be redundant/unecessary?

Reply:
seems ifdef script not need, game will not works completelu without sripts